labs-ios-starter/Login/View Controllers/LoginViewController.swift

```diff
@@ -61,13 +64,17 @@ class LoginViewController: DefaultViewController {
                 self.presentedViewController == nil else { return }
                 
             if exists {
-                self.performSegue(withIdentifier:
-                    .getSegueID(.showDetailProfileList), sender: nil)
+                self.handleLogin()
             } else {
-                self.performSegue(withIdentifier: .getSegueID(.modalAddProfile), sender: nil)
+                print("not logged in")
```

Comment:
Could use some safety in case of failure. Returning to login screen or something

Reply:
Good call, we'll have to handle the implementation details down the line. This gives us an obvious place to put the implementation